File: URI/Beginner/C/1012_Area.c
/*
	URI Online Judge Problem
	Category: Begginer
	Problem: 1012 - Area
	Status: Accepted
*/

#include <stdio.h>

#define PI 3.14159
#define square(x) x*x

int main() {
	double A, B, C;
	
	scanf("%lf %lf %lf", &A, &B, &C);
	
	printf("TRIANGULO: %.3f\n", A*C/2);
	printf("CIRCULO: %.3f\n", PI*square(C));
	printf("TRAPEZIO: %.3f\n", (A+B)*C/2);
	printf("QUADRADO: %.3f\n", square(B)); 
	printf("RETANGULO: %.3f\n", A*B);
	
	return 0;
}

File: URI/Beginner/Python/1012_Area.py
#=====================================    
#   URI Online Judge Problem
#   Category: Begginer
#   Problem: 1012 - Area
#   Status: Accepted
#=====================================

input = input()
input = input.split()

A = float(input[0])
B = float(input[1])
C = float(input[2])

PI = 3.14159

print('TRIANGULO: {:.3f}'.format(A*C/2))
print('CIRCULO: {:.3f}'.format(PI*C**2))
print('TRAPEZIO: {:.3f}'.format((A + B)*C/2))
print('QUADRADO: {:.3f}'.format(B**2))
print('RETANGULO: {:.3f}'.format(A*B))